src/element.js

```diff
@@ -36,7 +36,21 @@ function findElement(sendSessionCommand, strategy, selector) {
             getText: getText.bind(null, sendElementCommand),
             getCss: getCss.bind(null, sendElementCommand)
         };
-    });
+    }).catch(() => undefined);
+}
+
+async function findElement(sendSessionCommand, strategy, selector, maxWaitTime = 0) {
```

Comment:
It's a bit hard to decide. I can agree with not modifying the `findElement` but basically Selenium webdriver contains the `wait` functionality and the `until` helper library. 
I think the right endpoint would be the `POST timeouts` for this, but seems like Chromium not implemented it for example. As I've checked in Selenium's solution, they also wrote the timeout therefore in their JS implementation, but basically it's part of the webdriver package and the `wait` is definitely part of the driver's functionality.